setup.py

```diff
@@ -18,12 +18,12 @@
 setup(
     name="hashd",
     version=version,
-    description="A fully decentralized broadcast system for the based on gossip and proof of work",
```

Comment:
@Harryman Just so you know, that is copied from the README. I'll let you change it there.

Reply:
ah, didn't catch it in the README.md, i'll just edit here in github